platform/ui/src/components/AboutModal/AboutModal.tsx

```diff
@@ -73,71 +75,71 @@ const AboutModal = ({ buildNumber, versionNumber, commitHash }) => {
   );
   return (
     <div>
-      {renderRowTitle('Important Links')}
+      {renderRowTitle(t('Important links'))}
```

Comment:
Should we add `'Important links'` to some of the various AboutModal.json files? At least for en-US and test-LNG maybe?